File: ClassProject/Ex085Retrofit2/app/src/main/java/com/inah_wook/ex085retrofit2/Item.java
package com.inah_wook.ex085retrofit2;

public class Item {
    // 이 클래스는 우리가 가져올 json 파일의 데이터를 가진 객체로 만들고 싶어서 만든고얌.
    // 이 맴버 변수 이름들은 json 의 식별자면과 완전히 일치해야 함 !
    String title;
    String msg;

    public Item(String title, String msg) {
        this.title = title;
        this.msg = msg;
    }

    public Item() {
    }
}

File: ClassProject/Ex096KotlinViewTestInah/app/src/main/java/com/inah_wook/ex096kotlinviewtestinah/Item.kt
package com.inah_wook.ex096kotlinviewtestinah

data class Item constructor(var title:String ,var msg:String,var img:Int)
    // 주 생성자인 constructor 의 파라미터에 var 키워드를 넣으면 맴버변수가 됨
    // 약간 파이썬같넹 __init__()
    // 심지어....
    // 클래스에 내용 없으면 중괄호도 생략 가능함 ;

// data class  :  나중에 이 클래스의 객체를 두개 만들었을 때,
// String 의 .equals()로 비교하는거 할 수 있음 ~
// 나중에 객체 두개 만들어서 비교 가능



